src/main/java/net/sf/jabref/logic/importer/fileformat/MrDLibImporter.java

```diff
@@ -0,0 +1,261 @@
+/**
+ *
+ */
+package net.sf.jabref.logic.importer.fileformat;
+
+import java.io.BufferedReader;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import javax.xml.parsers.ParserConfigurationException;
+import javax.xml.parsers.SAXParser;
+import javax.xml.parsers.SAXParserFactory;
+
+import net.sf.jabref.logic.importer.Importer;
+import net.sf.jabref.logic.importer.ParserResult;
+import net.sf.jabref.logic.util.FileExtensions;
+import net.sf.jabref.model.database.BibDatabase;
+import net.sf.jabref.model.entry.BibEntry;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.xml.sax.Attributes;
+import org.xml.sax.SAXException;
+import org.xml.sax.helpers.DefaultHandler;
+
+/**
+ *
+ *
+ */
+public class MrDLibImporter extends Importer {
```

Comment:
I think `Parser` instead of `Importer` would be sufficient since you don't really need `isRecognizedFormat` (you more or less know that you get a valid format from your server)